src/main/java/com/core/model/dto/account/AccountRequestDto.java

```diff
@@ -0,0 +1,17 @@
+package com.core.model.dto.account;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Positive;
+import lombok.Data;
+
+@Data
+public class AccountRequestDto {
+    @Positive
+    private long accountNumber;
+    @NotBlank
+    @NotNull
+    private String currency;
```

Comment:
https://www.baeldung.com/javax-validations-enums

Reply:
I won't implement it for SB task. But for sure take it into account next time I need to validate enums. Thanks!